target/linux/pistachio/marduk/profiles/marduk.mk

```diff
@@ -25,7 +25,7 @@
 
 define Profile/marduk
     NAME:=Basic platform profile for Marduk
-    PACKAGES:=kmod-i2c kmod-usb-dwc2 kmod-marduk-cc2520 wpan-tools tcpdump uhttpd uboot-envtools
+    PACKAGES:=kmod-i2c kmod-usb-dwc2 kmod-marduk-cc2520 wpan-tools tcpdump uhttpd uboot-envtools alsa-lib alsa-utils alsa-utils-tests
```

Comment:
What kernel modules are required to support these packages? If they are not supported in OpenWRT (package/kernel) then these modules need to be added to modules.mk